service/src/main/java/org/apache/griffin/core/util/TimeUtil.java

```diff
@@ -148,4 +149,11 @@ private static String refreshEscapeHashTag(String str) {
         return str.replaceAll(escapeHashTagPattern, hashTag);
     }
 
+    public static TimeZone getTimeZone(String timezone) {
+        if (StringUtils.isEmpty(timezone)) {
+            return TimeZone.getDefault();
+        }
+        return TimeZone.getTimeZone(timezone);
+    }
+
 }
```

Comment:
LGTM, refactor to better place.

Reply:
@chemikadze 
I suggest this code be kept in its original place. Currently just modify ui code to fix bug.

Reply:
@ahutsunshine Well, it's about time, it's utility function, static makes it easier to test. Felt like it belongs here.

Reply:
yes, it is utility function.